Bug: Docker launch sim

when running `roslaunch pai sim.launch` in the Docker the launch fail

```
[ERROR] [1686156460.241993689, 1.435000000]: Exception thrown while initializing controller 'right_front_ctrl'.
Could not find resource 'right_front_joint' in 'hardware_interface::VelocityJointInterface'.
```

I thought it might be related with missing ROS depedencies in however even after installing all the dependencies in https://github.com/cychitivav/pai/blob/951ca9c63c6fe73660dc87310d84bdb785153710/Dockerfile. The problem still persisted.

--- a/Dockerfile
+++ b/Dockerfile
@@ -17,6 +17,9 @@ RUN apt-get install -y python3-pip
 # Install pigpio python module
 RUN pip3 install pigpio
 
+# Install ros control dependencies for simulation
+RUN apt-get install -y ros-noetic-ros-control ros-noetic-ros-controllers
+
 WORKDIR /root/catkin_ws/
 
 CMD ["bash"]

--- a/config/controller.yaml
+++ b/config/controller.yaml
@@ -4,18 +4,18 @@ joint_state_controller:
   publish_rate: 30
 
 # Velocity controllers
-left_back_ctrl:
+back_left_ctrl:
   type: velocity_controllers/JointVelocityController
-  joint: left_back_joint
+  joint: back_left_joint
 
-right_back_ctrl:
+back_right_ctrl:
   type: velocity_controllers/JointVelocityController
-  joint: right_back_joint
+  joint: back_right_joint
 
-left_front_ctrl:
+front_left_ctrl:
   type: velocity_controllers/JointVelocityController
-  joint: left_front_joint
+  joint: front_left_joint
 
-right_front_ctrl:
+front_right_ctrl:
   type: velocity_controllers/JointVelocityController
-  joint: right_front_joint
+  joint: front_right_joint

--- a/launch/sim.launch
+++ b/launch/sim.launch
@@ -28,7 +28,7 @@
     <!-- Load controller -->
     <node name="robot_controllers" pkg="controller_manager" type="spawner" respawn="false"
         output="screen"
-        args="joint_state_controller left_front_ctrl right_front_ctrl left_back_ctrl right_back_ctrl" />
+        args="joint_state_controller front_left_ctrl front_right_ctrl back_left_ctrl back_right_ctrl" />
 
     <!-- Node python scripts -->
     <node name="kinematic_model" pkg="pai" output="screen" type="kinematic_model.py" />

